extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -551,6 +557,8 @@ export class Bind {
     const applyPromises = this.boundElements_.map(boundElement => {
       const {element, boundProperties} = boundElement;
 
+      // TODO(choumx): We should avoid triggering a mutation if there are the
```

Comment:
Oops, fixed typo. 

I mean that on a page with 10 bindings and a state mutation only affects one of them, there will be 9 unnecessary measure/mutate passes scheduled that will early-out when checking if the expression result has changed. This seems pretty wasteful.